mythril/mythril.py

```diff
@@ -226,6 +256,23 @@ def set_api_rpc_localhost(self):
         self.eth = EthJsonRpc('localhost', 8545)
         logging.info("Using default RPC settings: http://localhost:8545")
 
+    def set_api_from_config_path(self):
+        config = ConfigParser(allow_no_value=False)
+        config.optionxform = str
+        config.read(self.config_path, 'utf-8')
+        if config.has_option('defaults', 'dynamic_loading'):
```

Comment:
You could change it to the following one liner:
```python
dynamic_loading = config.get('defaults', 'dynamic_loading') if config.has_option('defaults', 'dynamic_loading') else 'infura'
```

Reply:
Not that the current situation isn't good now :sweat_smile:

Reply:
But that would be quite a long 1 liner.